change/@azure-msal-browser-2020-10-21-13-05-39-msal-browser-storage-update.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "msal-browser and msal-node cache Interfaces to msal-common updated (#2415)",
+  "packageName": "@azure/msal-browser",
+  "email": "sameera.gajjarapu@microsoft.com",
+  "dependentChangeType": "patch",
```

I believe we want this set to `none` all the time, right @pkanher617 ?
```suggestion
  "dependentChangeType": "none",
```